cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -127,6 +194,74 @@ SPDX-License-Identifier: Apache-2.0
         font-size: 6px;
         line-height: 16px;
     }
+    ul.credential-list {
+        padding-left: 0;
+        display: flex;
+        flex-flow: row wrap;
+    }
+
+    li {
```

Comment:
you are changing `li` which is global, but the `<style>` is not `scoped`.

Reply:
Will make style scoped

Reply:
done